### PR TITLE
nuke openstack (again)

### PR DESCRIPTION
**Pull request by makefu**
###### Motivation for this change
openstack packaging in nixpkgs is a mess, not to mention the security issues it produces once people actually want to use it.
This PR tries to progressively remove the parts of openstack and tries to avoid conflicts.
Ref: #29491 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by makefu** [comment]
okay done

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -12747,840 +12600,162 @@ in {
     };
   };
 
-  oslosphinx = buildPythonPackage rec {
-    name = "oslosphinx-3.3.1";
+  rfc3986 = callPackage ../development/python-modules/rfc3986 { };
+
+   cachetools_1 = callPackage ../development/python-modules/cachetools/1.nix {};
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: indentation?

Comment (makefu) [reply pkgs/top-level/python-packages.nix]: weird, i fix this commit.

Comment (FRidh) [reply pkgs/top-level/python-packages.nix]: Actually, I think at some point the indentation is also incorrect in `python-packages.nix` already before these changes.

Comment (makefu) [reply pkgs/top-level/python-packages.nix]: Yes, i saw weird indentation in the `python-packages.nix` but i haven't touched it to keep the PR clean.

File: nixos/modules/misc/ids.nix [hunk 2]
```diff
@@ -281,8 +281,8 @@
       stanchion = 262;
       riak-cs = 263;
       infinoted = 264;
-      keystone = 265;
-      glance = 266;
+      # keystone = 265; # unused
+      # glance = 266; #unused
```
Comment (Mic92) [inline nixos/modules/misc/ids.nix]: We usually use `unused` when only either group id or user id is unused.

Comment (makefu) [reply nixos/modules/misc/ids.nix]: Oh, i thought it was for showing that there are still `unused` numbers from old packages in this sections which are up for grabs. i can keep 2 empty lines instead

Comment (Mic92) [reply nixos/modules/misc/ids.nix]: ok, then please that remark.

Comment (makefu) [reply nixos/modules/misc/ids.nix]: okay done

**Comment by Mic92** [comment]
Thanks!

**Comment by makefu** [comment]
thanks! maybe a short remark should be added to the release notes

**Pull request merged**